### PR TITLE
Check and log transactions (RT-3499 - RT-3501)

### PR DESCRIPTION
**Pull request by h0vhannes**

### CODE REVIEW

**Comment by vhpoet** [comment]
LGTM

File: src/js/services/api.service.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  rpAPI.getSubscription = function() {
+    return $http.get(Options.backend_url + '/api/subscription', httpOptions);
+  };
+
+  rpAPI.updateSubscription = function(subscription) {
+    return $http.put(Options.backend_url + '/api/subscription', subscription, httpOptions);
+  };
+
+  rpAPI.addTransaction = function(transaction, result, hash, local_time) {
```
Comment (vhpoet) [inline src/js/services/api.service.js]: Wouldn't `logTransaction` be a better name for this?

**Comment by vhpoet** [comment]
The naming convention for angular dependencies is
- prepend `$` if the dependency is an angular service
- don't prepend `$` if the dependency is a custom service

so `$scope, $routeParams, api, id, network`

File: src/js/services/api.service.js (hunk 1 above)
Comment (h0vhannes) [inline src/js/services/api.service.js]: add is used to keep CRUD logic, log sounds better for sure.

**Comment by h0vhannes** [comment]
Totally agree with this one. I was just confused because this convention is not preserved everywhere, so that's why I used $api in a number of places. But I'll change that.

**Comment by h0vhannes** [comment]
@vhpoet - fixed, please have a look

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**